custom_components/ai_thermostat/models/utils.py

```diff
@@ -1,8 +1,6 @@
-import asyncio
 import logging
-from datetime import datetime, timedelta
 
-from custom_components.ai_thermostat.helpers import convert_decimal, set_trv_values
+from custom_components.ai_thermostat.helpers import convert_decimal
```

Comment:
This is not part of the PR's intended changes. Please remove this from your changeset :)